kubernetes/deployment.tmpl

```diff
@@ -82,6 +100,19 @@ metadata:
 spec:
   selector:
     app: caesar-staging-app
+  ports:
+    - protocol: TCP
+      port: 81
+      targetPort: 80
+  type: NodePort
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
```

Comment:
This should really be called caesar-staging-nginx. Since it's already deployed, you'll need to `kubectl delete service/nginx` after changing it. 

Also the ingress config in the ops repo needs updating since it's still pointing to caesar-staging-app.

Reply:
Actually, no wait. Maybe this nginx service shouldn't be needed at all and the caesar-staging-app service shouldn't have been changed?

Reply:
I'm not sure it's clear what I mean, so I've opened #752.